views/js/qtiCreator/model/variables/ResponseDeclaration.js

```diff
@@ -13,8 +13,8 @@ define([
     _.extend(methods, editable);
     _.extend(methods, {
         setTemplate : function(template){
-            var templateUri = responseHelper.getTemplateUriFromName(template);
-            if(templateUri && this.template !== templateUri){
+            var templateUri = responseHelper.getTemplateUriFromName(template) || null;
+            if(this.template !== templateUri){
```

Comment:
Why was removed condition `templateUri && `?

Reply:
because for custom response type there is no `templateUri` so it will be changed to `null`